### PR TITLE
Remove publications frontend code

### PR DESCRIPTION
**Pull request by mgrassotti**
https://trello.com/c/kfyYY4tf/368-9-publications-migration-final-tasks-deploy-2-0-5-days

Remove any publications frontend code in Whitehall that cannot be fact-checked

### CODE REVIEW

**Comment by mgrassotti** [comment]
I had to remove 
```
features/admin-broken-link-reporting.feature
admin-highlight-words-to-avoid.feature
``` 
to make tests pass but to be honest I am not sure why they were failing.

File: app/controllers/publications_controller.rb [hunk 1]
```diff
@@ -1,54 +0,0 @@
-class PublicationsController < DocumentsController
-  enable_request_formats index: [:json, :atom]
-  before_filter :expire_cache_when_next_publication_published
-  before_filter :redirect_statistics_filtering, only: [:index]
-  before_filter :redirect_statistics_documents, only: [:show]
-
-  def index
```
Comment (gpeng) [inline app/controllers/publications_controller.rb]: We can't remove this as I think we'll still need to provide the `.atom` feed of publications.

**Comment by fofr** [comment]
What's the status of this?

**Comment by mgrassotti** [comment]
@fofr @gpeng This is ready to be reviewed again

**Comment by gpeng** [comment]
It needs preview to be switched to government-frontend before we can merge this. I've published all of the draft links but there are still some issues with the sync checks.

**Comment by rubenarakelyan** [comment]
Closing this until we're ready.

**Comment by fofr** [comment]
Is there a list of tickets showing the frontend code that's still in Whitehall for each of the migrated formats that hasn't yet been deleted?

**Comment by rubenarakelyan** [comment]
@fofr Not that I know of - I've taken a look in the whitehall support planning board.

**Comment by tuzz** [comment]
Superseded by: https://github.com/alphagov/whitehall/pull/3370

**Pull request closed without merging**